### PR TITLE
Add a template for deprecation notice

### PR DESCRIPTION
**Pull request by VickyAtAZ**
## What does this PR do?

Add a template for deprecation notice. This template is used for create a github issue which is mentioned [here ](https://github.com/o3de/community/blob/main/guides/o3de-deprecation-guidelines.md#api-deprecation---initial-steps)

## How was this PR tested?

md file viewer

image

### CODE REVIEW

**Comment by lmbr-pip** [comment]
Can we fix title so its clear what this?

**Comment by VickyAtAZ** [comment]
@lmbr-pip The O3DE deprecation guide can be found here: https://github.com/o3de/community/blob/main/guides/o3de-deprecation-guidelines.md which should answer most of your questions.  @hultonha Tom would be the best to answer any other questions you may have.

This template is only for the small change mentioned in that document:
"For a small change (e.g. spelling/grammar/formatting), please create a [GitHub issue](https://github.com/o3de/sig-content/issues/new/choose) or [pull request](https://github.com/o3de/sig-content/pulls) against the [Community repo](https://github.com/o3de/community) with the fix)."

**Review by hultonha — APPROVED** [review]
This looks great to me, awesome work @VickyAtAZ 👍

File: .github/ISSUE_TEMPLATE/deprecation_template.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+---
+
+## Deprecated APIs
+List the fully qualified APIs/types. For example, `AZ::Quaternion::SetFromEulerRadians(const Vector3&)`
+
+## Alternatives APIs
+List new or updated APIs to replace deprecated API. 
+For example, use 
+`Quaternion AZ::Quaternionstatic CreateFromEulerRadiansXYZ(const Vector3&)`
```
Comment (lumberyard-employee-dm) [inline .github/ISSUE_TEMPLATE/deprecation_template.md:15]: This declaration is smushed together.

**Review by lumberyard-employee-dm — APPROVED** [review]

**Pull request merged**